### PR TITLE
Applying rules for first time

### PR DESCRIPTION
**Pull request by mconnell**
This PR introduces a flow for adding new rules after creating a new feed. The flow after creating a new feed is slightly different to an existing feed, as the user should be presented with all of the source rules check by default, with the option to uncheck them.

### CODE REVIEW

File: app/views/admin/rules/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<h1>Apply Source Rules</h1>
+
+<%= form_tag admin_partner_feed_rules_path(@partner, @feed), method: :post do |form| %>
```
Comment (munckymagik) [inline app/views/admin/rules/new.html.erb]: Admittedly, I'm still feeling my way with the Dough stuff, should this be a Dough form builder thingey?

**Comment by munckymagik** [comment]
:+1:

File: app/views/admin/rules/new.html.erb (hunk 1 above)
Comment (Tr4pSt3R) [inline app/views/admin/rules/new.html.erb]: <div id="calc">
  <%= form_for @calculator, url: { action: "create" }, html: { class: 'form' }, builder: Dough::Forms::Builders::Validation do |f| %>
    <%= f.validation_summary %>
.
.
.
</div>

Yes, you may need something like the above to make forms Doughey. 

Example file:
(https://github.com/moneyadviceservice/cutback-calculator/blob/master/app/views/cutback_calculator/calculators/new.html.erb)

**Comment by rnalexander** [comment]
:+1: :shipit:

**Comment by mconnell** [comment]
@Tr4pSt3R @munckymagik dough'd the forms, thanks.

**Comment by munckymagik** [comment]
:100: :+1:

**Pull request merged**